packages/demo-saas/next.config.js

```diff
@@ -0,0 +1,7 @@
+const nextConfig = {
+  experimental: {
+    appDir: true,
+  },
```

Comment:
I prefer to create two demo-saas application one for nextjs12 and one for nextjs13